src/main/kotlin/nl/info/zac/app/zaak/converter/RestZaakConverter.kt

```diff
@@ -216,4 +223,8 @@ class RestZaakConverter @Inject constructor(
             ?.forEach(gerelateerdeZaken::add)
         return gerelateerdeZaken
     }
+
+    private fun isOntvangstbevestigingNietVerstuurdIndicationSetRequired(zaak: Zaak, statustype: StatusType?) =
```

Comment:
```suggestion
    private fun isOntvangstbevestigingNietVerstuurdIndicationSetRequired(zaak: Zaak, statustype: StatusType) =
```
?

Or can it really be null?

Also the function name is really unclear to me personally (and to the AI as well I guess; which also needs to be able to read our code).